.evergreen/config.in.yml

```diff
@@ -753,6 +767,7 @@ functions:
           NODE_VERSION: ${NODE_VERSION}
           MONGODB_URI: ${MONGODB_URI}
         binary: bash
+        add_expansions_to_env: true
```

Comment:
do we know whether this would set or override any of the env vars defined above?